### PR TITLE
Add ColorPicker and Handle components

### LINKED ISSUE #2: Create color picker component
- [x] Color wheel should be a correct representation of a hue wheel.
- [x] It should have two handles that can be dragged around.
- [x] When handles are dragged it updates selected colors.

### PR DESCRIPTION
**Pull request by adamxberlin**
Partial implementation of #2

### CODE REVIEW

File: src/components/handle.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    radianAngle: 0,
+  };
+
+  [handleRotation](e: MouseEvent) {
+    if (!this.state.isMouseDown) return;
+    if (!this.container.current) return;
+
+    const boundingRect = this.container.current.getBoundingClientRect();
+
+    const delta = {
```
Comment (sunyatasattva) [inline src/components/handle.tsx:29]: Yes! Love the way you cleaned that up 👍

File: src/components/handle.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import '../styles/handle.css';
+import React from 'react';
+
+const handleRotation = Symbol();
+const toggleMouseDown = Symbol();
```
Comment (sunyatasattva) [inline src/components/handle.tsx]: Why would you use `Symbol`s here? It really seems like a completely forced, overly-clever solution, to the wrong problem. Unless I'm missing something, there is no reason to use those over normal method names.

If you wanted privacy, note that `Symbol`s are not private. If you wanted to denote privacy, Typescript allows you to have `private` methods that make that explicit. Or, if you wanted, you could use the official `#method()` experimental proposal through Babel. But I would just stay with TS.

**Review by sunyatasattva — COMMENTED** [review]
Everything besides my one comment about `Symbol` looks great! However, this does not technically close #2 as you have written in your comment, because #2 is the whole Color Picker component, which we don't yet have.

File: src/components/handle.tsx (hunk 1 above)
Comment (adamxberlin) [reply src/components/handle.tsx:29]: Thx 🙏😄

File: src/components/handle.tsx (hunk 2 above)
Comment (adamxberlin) [reply src/components/handle.tsx]: https://medium.com/@martin_hotell/10-typescript-pro-tips-patterns-with-or-without-react-5799488d6680 ⬅️Bullet point 3

Just trying out new patterns 😬 But I agree, it feels a bit convoluted... We can change it back to private to be more explicit if you want. Come to think of it, I think I'd prefer to actually.

I mean, the private accessor is just to help us avoid mistakes while coding in this case . Everything on runtime happens client-side, so we don't really care if the end user messes around with the code.

**Comment by adamxberlin** [comment]
> Everything besides my one comment about `Symbol` looks great! However, this does not technically close #2 as you have written in your comment, because #2 is the whole Color Picker component, which we don't yet have.

You're right! Changed the comment to "Partial implementation of #2" so it's more correct 👍

File: src/components/handle.tsx (hunk 2 above)
Comment (adamxberlin) [reply src/components/handle.tsx]: Fixed and pushed it to the PR branch 👍

**Pull request merged**